.npmrc

```diff
@@ -1,3 +1,2 @@
 registry=https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public-npm/npm/registry/
-cache=./src/submodules/Node-Externals/cache
```

Comment:
What does that mean for people running `npm install` in local development? Is that going to fail or behave strangely since they don't have this environment variable?

Reply:
Would it make sense to specify it in both `.npmrc` and the env var so it works in both cases?

Reply:
We can't specify it here and override it with an environment variable. The config file wins over the environment variable

> What does that mean for people running npm install in local development? Is that going to fail or behave strangely since they don't have this environment variable?

It will work but just use the global cache. It'll fail in CI if it updated the package-lock.json file as the local cache won't be updated. That's why we are working on automation to enable updating deps directly from CI or running a script locally, so that it takes care of making all the commits and creating all the relevant PRs automatically.

The problem with the .npmrc file is that it only accepts a relative path, so if you run the command from somewhere else outside of the root, it makes a mess.

Reply:
> It will work but just use the global cache

Sounds fine then. Thanks!